docs/source/examples/mg-mode-part-iii.ipynb

```diff
@@ -0,0 +1,1460 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we must specify to the Library which types of cross sections to compute. OpenMC's multi-group mode can accept isotropic flux-weighted cross sections or angle-dependent cross sections, as well as supporting anisotropic scattering represented by either Legendre polynomials, histogram, or tabular angular distributions.  \n",
+    "\n",
+    "Just like before, we will create the following multi-group cross sections needed to run an OpenMC simulation to verify the accuracy of our cross sections: \"total\", \"absorption\", \"nu-fission\", '\"fission\", \"nu-scatter matrix\", \"multiplicity matrix\", and \"chi\".\n",
+    "\"multiplicity matrix\" is needed to provide OpenMC's multi-group mode with additional information needed to accurately treat scattering multiplication (i.e., (n,xn) reactions)) explicitly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Specify multi-group cross section types to compute\n",
+    "iso_mgxs_lib.mgxs_types = ['total', 'absorption', 'nu-fission', 'fission',\n",
+    "                           'nu-scatter matrix', 'multiplicity matrix', 'chi']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we must specify the type of domain over which we would like the `Library` to compute multi-group cross sections. The domain type corresponds to the type of tally filter to be used in the tallies created to compute multi-group cross sections. At the present time, the `Library` supports \"material\" \"cell\", \"universe\", and \"mesh\" domain types. \n",
+    "\n",
+    "For the sake of example we will use a mesh to gather our cross sections. This mesh will be set up so there is one mesh bin for every pin cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Instantiate a tally Mesh\n",
+    "mesh = openmc.Mesh()\n",
+    "mesh.type = 'regular'\n",
+    "mesh.dimension = [10, 10]\n",
+    "mesh.lower_left = [0., 0.]\n",
+    "mesh.upper_right = [length, length]\n",
+    "\n",
+    "# Specify a \"mesh\" domain type for the cross section tally filters\n",
+    "iso_mgxs_lib.domain_type = \"mesh\"\n",
+    "\n",
+    "# Specify the mesh over which to compute multi-group cross sections\n",
+    "iso_mgxs_lib.domains = [mesh]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will set the scattering treatment that we wish to use.\n",
+    "\n",
+    "In the `mg-mode-part-ii` notebook, the cross sections were generated with a typical P3 scattering expansion in mind. Now, however, we will use a more advanced technique: OpenMC will directly provide us a histogram of the change-in-angle (i.e., $\\mu$) distribution.\n",
```

Comment:
Yep! markdown for the win! (It wont work in the actual notebook hosted live by Jupyter, but it will on our docs)